server/vcr-server/vcr_server/settings.py

```diff
@@ -173,6 +173,10 @@ def parse_bool(val):
     ],
 }
 
+CRED_TYPE_SYNONYMS = {
+    "registration": "registration.registries.ca"
+}
+
```

Comment:
Do we need to have this in a more "configurable" spot for future use (e.g.: an external mappings file) or am I worrying too soon about something we may not really have to worry about?

Reply:
Possibly, I just wanted to get in something quick & easy for now

Reply:
Another option is to query an environment variable (would be consistent with the other configs)

Reply:
Sounds good. We don't even know if this will be a recurring issue so having this and building on top of it seems fine.